skietbaan_fe/src/components/RegisterMemberStyles.css

```diff
@@ -1,46 +1,41 @@
 @media all and (min-width: 480px) {
-    .Register {
-      padding: 60px 0;
+    .registermember {
+      margin: 0 auto;
     }
-  
-    .Register form {
+    .registermember form {
       margin: 0 auto;
-      max-width: 320px;
     }
   }
-  
   body{
     background-color:red;
   }
-  
-  .test{
-    width:50%
-  }
   .col-1,.col-2,.col-3,.col-4,.col-5,.col-6,.col-7,.col-8,.col-9,.col-10,.col-11,.col-12{
     float:left;
-  
   }
-  .centre-register{
-    width:94%;
+  .centre-register-member{
     margin:0 auto;
     position: absolute;
-    left: 3%;
-    top: 15%;
+    left: 3px;
```

Comment:
if you have margin auto you don't need the left, it's just going to move it off center by 3 pixels